message_tools/find_invalid_address_case_id_by_uprn.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import csv
+from pathlib import Path
+
+import requests
+from google.cloud import storage
+from requests import HTTPError
+
+from config import Config
+
+
+def read_invalid_address_uprn_file(file_to_process, project_id):
```

Comment:
This function is doing a lot more than reading the file, name should reflect that it is generating a bulk invalid address file.